examples/app.js

```diff
@@ -261,6 +265,67 @@ async function logLevelSelected(event) {
     });
 }
 
+$('#wssUrl-button').click(async () => {
+    const form = $('#wssForm');
+    if (!form[0].checkValidity()) {
+        return;
+    }
+    ROLE = 'viewer';
+
+    const localView = $('#viewer .local-view')[0];
+    const remoteView = $('#viewer .remote-view')[0];
+    const localMessage = $('#viewer .local-message')[0];
+    const remoteMessage = $('#viewer .remote-message')[0];
+    const sourceURL = $('#sourceURL').val();
+    const formValues = {
+        region: $('#wssRegion').val(),
+        channelName: $('#wssChannelName').val(),
+        }
+    const authToken = $('#wssAuthToken').val().replace(/\s+/g, '');
+    /*
+    if (formValues.enableDQPmetrics) {
+        $('#dqpmetrics').removeClass('d-none');
+        $('#webrtc-live-stats').removeClass('d-none');
+    }
+    toggleDataChannelElements();
+    */
+    $(remoteMessage).empty();
+    localMessage.value = '';
+
+    const targetUrl = `${sourceURL}/api/video-services/video-call/connect`;
```

Comment:
Should we remove the specific path to keep this app independent?

Reply:
I was considering that, too, but I didn't know how brivo free we wanted to make it. I'll probably switch it so that the URL has the entire local URL as the default text entry but allow the user to plop in whatever they'd like